README.rst

```diff
@@ -125,6 +129,9 @@ If you have more than one role available to you, you'll be prompted to
 choose the role from a list; otherwise, if your credentials are correct,
 you'll just see the AWS keys printed on stdout.
 
+If you're using Docker with a U2F security key (Yubikey or similar) you won't
```

Comment:
The sentence is awkward,  perhaps you meant to say `If you're using a U2F security key with your Google account you won't be ...`

Can you expand on what `Docker/Host isolation` mean?